openc3/lib/openc3/microservices/reaction_microservice.rb

```diff
@@ -285,29 +281,52 @@ def run_reaction(reaction:)
     end
 
     def run_action(reaction:, action:)
+      reaction_json = reaction.as_json(:allow_nan => true)
+      # Let the frontend know which action is being run
+      # because we can combine commands and scripts with notifications
+      reaction_json['action'] = action['type']
+      notification = {
+        'kind' => 'run',
+        'type' => 'reaction',
+        'data' => JSON.generate(reaction_json),
+      }
+      AutonomicTopic.write_notification(notification, scope: @scope)
```

Comment:
There previously was no indication of when an action was run so I added a notification to the AutonomicTopic